modules/oe_content_publication/config/install/core.entity_view_display.node.oe_publication.default.yml

```diff
@@ -28,24 +87,71 @@ content:
       link: false
     third_party_settings: {  }
     region: content
+  oe_publication_contact:
```

Comment:
We don't need oe_publication_contact anymore

Reply:
This is removed.